scanners/zap/zap_none.py

```diff
@@ -213,8 +200,50 @@ def _add_env(self, key, value=None):
     # Accessed by ZapNone only                                    #
     # + MUST be implemented                                       #
     ###############################################################
+    def _handle_plugins(self):
```

Comment:
wondering if it's worth implementing common parts of  _handle_plugins() in zap.py?

Reply:
yes, good point. done